src/Sources/Linq2Acad/Extensions/GroupExtensions.cs

```diff
@@ -0,0 +1,123 @@
+﻿using Autodesk.AutoCAD.DatabaseServices;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Linq2Acad
+{
+    /// <summary>
+    /// Extension records for instance of Group.
+    /// </summary>
+    public static  class GroupExtensions
+    {   /// <summary>
+        /// Appends ids to group. (Note, this crystalises the ids to an array internally).
+        /// </summary>
+        /// <param name="group"></param>
+        /// <param name="ids"></param>
+        public static void Append(this Group group, IEnumerable<ObjectId> ids)
+        {
+            ObjectIdCollection idCollection = new ObjectIdCollection(ids.ToArray());
```

Comment:
ObjectIdCollections have to be disposed of. Best way to reliably call Dispose() is a using statement:

```cs
using (ObjectIdCollection idCollection = new ObjectIdCollection(ids.ToArray()))
{
  group.Append(idCollection);
}
```

Not disposing of ObjectIdCollections introduces a memory leak (and AutoCAD crashes unpredictably).

Reply:
Neither did I. Was one of the many surprises the AutoCAD API can give you. ;) I once did a larger batch process with some custom code and AutoCAD crashed unpredictably. Calling Dispose() on the ObjectIdCollection fixed the problem.